site/scaling.md

```diff
@@ -151,7 +150,7 @@ see [Configuring Policies and Actions](https://docs.oracle.com/middleware/1221/w
 
     * `cluster_name` - WebLogic cluster name (Required)
 
-    * `kubernetes_master` - Kubernetes master URL, default=https://kubernetes
+    * `kubernetes_master` - Kubernetes master URL, default=https://kubernetes.  Note: set this to https://kubernetes.default.svc when invoking scalingAction.sh from the AdminisrationServer pod.
```

Comment:
Note: set this to https://kubernetes.default.svc when invoking scalingAction.sh from the AdminisrationServer pod. -> **Note**: Set this to `https://kubernetes.default.svc` when invoking `scalingAction.sh` from the Administration Server pod.

Reply:
Fixed